src/core/randomizer.js

```diff
@@ -1,37 +1,31 @@
-import { messages } from "./messages";
-
-function buildRandomizer() {
+function buildRandomizer(messages) {
   const state = {
-    currentMessage: messages[0],
+    messages: [...messages],
```

Comment:
I think we can just do something like this:

```suggestion
    messages,
```

Reply:
Alright 👍